jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/VehicleSizeBuilder.java

```diff
@@ -48,14 +48,14 @@ public static J2735VehicleSize genericVehicleSize(JsonNode vehicleSize) {
         if (length != 0) {
             genericVehicleSize.setLength(length);
         } else {
-            genericVehicleSize.setLength(null);
+            genericVehicleSize.setLength(0);
```

Comment:
If length == 0, the else statement is executed. Do we still need this if else statement at all?

Reply:
this is a solid point

Reply:
Addressed